packages/dropdown/src/components/Dropdown.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import Button, { buttonSizes } from "@crave/farmblocks-button";
+import {
+  Button as AriaButtonWrapper,
+  Wrapper as AriaWrapper,
+  Menu
+} from "react-aria-menubutton";
+
+import DropdownWrapper from "../styledComponents/DropdownWrapper";
+import DropdownMenuWrapper from "../styledComponents/DropdownMenuWrapper";
+
+const Dropdown = props => (
+  <DropdownWrapper>
+    <AriaWrapper onSelection={props.handleSelection}>
+      <AriaButtonWrapper>
+        <Button
+          rightIcon="wg-small-arrow-bottom"
+          text={props.text}
+          size={buttonSizes.MEDIUM}
```

Comment:
What about to receive `leftIcon` and `rightIcon`? Set default `rightIcon` to  `wg-small-arrow-bottom`?

Reply:
I dont know... I was thinking about the ability to receive any node as trigger (in the future) and not only properties for a button...

but if you want to attack the case linked above, leftIcon and rightIcon could work, and you will need an extra one for button size, because it looks smaller